app/assets/javascripts/discourse/app/components/topic-footer-buttons.gjs

```diff
@@ -151,7 +155,7 @@ export default class TopicFooterButtons extends Component {
               @id={{concat "topic-footer-dropdown-" actionable.id}}
               @value={{actionable.value}}
               @content={{actionable.content}}
-              @onChange={{action actionable.action}}
+              @onChange={{bind actionable.action this}}
```

Comment:
This is a weird one... but plugins are relying on `this.` being a reference to the TopicFooterButtons component instance 😬